### ISSUE TITLE
Generating png with transparent background from eps logo

### ISSUE THREAD

**Issue by egpotter** [python-pillow/Pillow#2164]
### What did you do?

Tryied to generate a .png with transparent background from an .eps logo.
### What did you expect to happen?

I expected the background to be transparent. I was able to do that with imagemagick easily, but I couldn't figure out a way to do it with Pillow.
### What actually happened?

The background is always white.
### What versions of Pillow and Python are you using?

Python 3.4.5 and Pillow 3.3.3
Please include code that reproduces the issue and whenever possible, an image that demonstrates the issue. The best reproductions are self-contained scripts with minimal dependencies. If you are using a framework such as plone, django, or buildout, try to replicate the issue just using Pillow. 

``` python
from PIL import Image


file_name = '/home/eduardo/Downloads/Volkswagen_logo.eps'
out_file = 'test.png'

image = Image.open(file_name)
image.save(out_file)
```

I also tried using mask for adding an alpha channel, but could not achieve this.

**Comment by wiredfool** [comment]
You should be able to add an alpha mask with <a href="http://pillow.readthedocs.io/en/3.4.x/reference/Image.html#PIL.Image.Image.putalpha">Image.putalpha</a>.  Note that the alpha mask needs to be a mode `1` or `L` image of the same size as the original image.  After you do that, the image mode should be `RGBA`, at which point you'll have a transparent image when you save in png format. 

The code that you have posted is likely getting an RGB image from the eps file, then saving that back out in PNG format. (but it's hard to tell without actually seeing the image)

**Comment by egpotter** [comment]
I have already tried adding the alpha via putalpha. The code should work with any eps image without background. 

```python
from PIL import Image


file_name = '/home/eduardo/Downloads/Volkswagen_logo.eps'
out_file = 'test.png'

image = Image.open(file_name)
imgSize = image.size
mask=Image.new('L', image.size, color=255)
image.putalpha(mask)
image.save(out_file)

```
Either I get a white background, or a transparency that is also applied to the image (logo) itself. 

This is the [logo](http://files.all-free-download.com/downloadfiles/graphic/graphic_1/volkswagen_logo_31115.zip ) I'm tring to open and convert to png via pillow. Maybe I'm being really stupid at some point, but I already spent some time into it and could not solve this.
